release/models/containerz/openconfig-containerz.yang

```diff
@@ -0,0 +1,139 @@
+module openconfig-containerz {
+  yang-version 1;
+  namespace "https://github.com/openconfig/yang/gnoi/containerz";
+  prefix oc-gnoi-containerz;
+
+  import openconfig-extensions { prefix oc-ext; }
+  import openconfig-system {
+    prefix oc-sys;
+  }
+  import openconfig-system-grpc {
+    prefix oc-sys-grpc;
+  }
+  import openconfig-types {
+    prefix oc-types;
+  }
+  import openconfig-yang-types {
+    prefix oc-yang;
+  }
+  organization
+    "OpenConfig Working Group";
+
+  contact
+    "OpenConfig working group
+    netopenconfig@googlegroups.com";
+
+  description
+    "This module provides a data model for the metadata of gNOI.containerz
+    running on this system as well as the containers running on the system.";
+
+  oc-ext:openconfig-version "0.1.0";
+
+  revision 2025-11-09 {
+    description
+      "Initial revision.";
+    reference "0.1.0";
+  }
+
+
+
+  // gRPC server related definitions.
+  // Success/failure counters.
+  grouping grpc-server-containerz-counters  {
+    description
+      "A collection of counters that were collected while evaluating
+      access to the gRPC server.";
+
+    container counters {
```

Comment:
This appears to try to add an overlapping container under `/system/grpc-servers/grpc-server/state`.  Suggest consideration of bucketizing "containerz" specific state/counters under it's own hierarchy specific to this set of services